src/main/java/core/basesyntax/data/FormaterImpl.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.data;
+
+import core.basesyntax.operation.Operation;
+import core.basesyntax.transaction.FruitTransaction;
+import java.util.Arrays;
+import java.util.List;
+
+public class FormaterImpl implements Formater {
```

Comment:
The class name 'FormaterImpl' seems to be a typo. The correct spelling would be 'FormatterImpl', with double 't'. Consider renaming the class to follow the naming conventions.